Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml

```diff
@@ -2359,3 +2359,39 @@
     seedId: cherry
   - type: SpaceGarbage
   - type: BadFood
+
+- type: entity
+  name: anomaly berry
+  description: a strange blue fruit. Something about it doesn't seem right.
+  parent: FoodProduceBase
+  id: FoodAnomalyBerry
+  components:
+  - type: FlavorProfile
+    flavors:
+      - strange
+  - type: Sprite
+    sprite: Objects/Specific/Hydroponics/anomaly_berry.rsi
+    state: produce
+  - type: Produce
+    seedId: anomalyBerry
+  - type: Food
+    trash:
+    - EffectAnomalyFloraBulb # Random loot
```

Comment:
That spawner hasn't been updated in ages. It should also be converted to use entityTable instead of a RandomSpawner. There is also an existing spawner in food_produce.yml, but that one is also out of date.

Reply:
I guess that cleanup can be done in a separate PR, I'll make one in the next few days.